content/policy/PolicyController.js

```diff
@@ -18,13 +18,48 @@ function($scope, $anchorScroll, $routeParams, loginStatusProvider, eventHandler,
 	$scope.closeAlert = function(){ $scope.alert = null; };
 	$scope.UserAuthenticated = false;
 	$scope.links = [];
+	
 	function SetupUser() {
 		return loginStatusProvider.validateAuthenticationPromise()
```

Comment:
The only thing we need to do inside the validateAuthenticationPromise.then(function()) is to call
```
$scope.$apply(function() { $scope.UserAuthenticated = true; });
```
everything else doesn't apply to this screen, right?